pennylane_forest/converter.py

```diff
@@ -1,7 +1,5 @@
 import warnings
```

Comment:
In this file there is a line which uses the `pyquil.simulation` submodule:
```python
_matrix_dictionary = pyquil.simulation.matrices.QUANTUM_GATES
```

For some reason this now crashes the tests. I realised that in the current pyquil version, `simulation` is not imported top level, that is why Python can't find it.

This can be solved by importing simulation directly:
```python
import pyquil.simulation as simulation
```